client/my-sites/controller.js

```diff
@@ -113,30 +128,39 @@ function renderNoVisibleSites( context ) {
 }
 
 function renderSelectedSiteIsDomainOnly( reactContext, selectedSite ) {
-	const EmptyContentComponent = require( 'components/empty-content' );
+	const FeatureUnavailable = require( 'components/empty-content/feature-unavailable' );
 	const { store: reduxStore } = reactContext;
 
-	removeSidebar( reactContext );
+	renderWithReduxStore( (
+			<FeatureUnavailable selectedSite={ selectedSite } />
+		),
+		document.getElementById( 'primary' ),
+		reduxStore
+	);
 
 	renderWithReduxStore(
-		React.createElement( EmptyContentComponent, {
-			title: i18n.translate( 'Add a site to start using this feature.' ),
-			line: i18n.translate( 'Your domain is only set up with a temporary page. Start a site now to unlock everything WordPress.com can offer.' ),
-			action: i18n.translate( 'Create Site' ),
-			actionURL: `/start/site-selected/?siteSlug=${ encodeURIComponent( selectedSite.slug ) }&siteId=${ encodeURIComponent( selectedSite.ID ) }`,
-			secondaryAction: i18n.translate( 'Manage Domain' ),
-			secondaryActionURL: domainManagementList( selectedSite.slug )
-		} ),
-		document.getElementById( 'primary' ),
+		createNavigation( reactContext ),
+		document.getElementById( 'secondary' ),
 		reduxStore
 	);
 }
 
 function isPathAllowedForDomainOnlySite( pathname, domainName ) {
 	const urlWhiteListForDomainOnlySite = [
-		domainManagementList( domainName ),
-		domainManagementEdit( domainName ),
+		domainManagementAddGoogleApps( domainName, domainName ),
+		domainManagementContactsPrivacy( domainName ),
 		domainManagementDns( domainName ),
+		domainManagementEdit( domainName ),
+		domainManagementEditContactInfo( domainName ),
+		domainManagementEmail( domainName, domainName ),
+		domainManagementEmailForwarding( domainName ),
+		domainManagementList( domainName ),
+		domainManagementNameServers( domainName ),
+		domainManagementPrivacyProtection( domainName ),
+		domainManagementRedirectSettings( domainName ),
+		domainManagementTransfer( domainName ),
+		domainManagementTransferOut( domainName ),
+		domainManagementTransferToAnotherUser( domainName ),
 		`/checkout/${ domainName }`,
```

Comment:
It seemed to be the case across the codebase, what would you suggest?

![screen shot 2017-02-22 at 22 14 28](https://cloud.githubusercontent.com/assets/326402/23230372/59002562-f94c-11e6-8fd4-e4ffb9c69c60.png)